server/internal/server/vector_store_files.go

```diff
@@ -254,6 +254,11 @@ func (s *S) DeleteVectorStoreFile(
 		return nil, err
 	}
 
+	if err := s.embedder.DeleteFile(ctx, req.VectorStoreId, req.FileId); err != nil {
+		// milvus does not return error if the file does not exist.
```

Comment:
Could you add a TODO comment on handling a not-found error gracefully? I think that's needed to handle a case where this RPC fails between `embedder.DeleteFile` and `store.DeleteFile` and allow retry.